nestkernel/event_delivery_manager.cpp

```diff
@@ -24,11 +24,7 @@
 
 // C++ includes:
 #include <algorithm> // rotate
-#include <iostream>
-#include <numeric> // accumulate
-
-// Includes from libnestutil:
-#include "logging.h"
+#include <numeric>   // accumulate
```

Comment:
This belongs under C++ includes above. The header from libnestutil is `numerics.h`.

Reply:
It actually is under C++ includes, see https://github.com/nest/nest-simulator/blob/28e42a40e6814fe248059be6c1a0e59b631555a1/nestkernel/event_delivery_manager.cpp#L27
It just looks like it's under libnestutil in the changelist, but the whole libnestutil block is actually gone.

Reply:
Sorry, my oversight!